projects/igniteui-angular/src/lib/splitter/splitter-pane/splitter-pane.component.ts

```diff
@@ -137,10 +152,11 @@ export class IgxSplitterPaneComponent {
      */
     @HostBinding('style.flex')
     public get flex() {
-        const grow = this.size !== 'auto' ? 0 : 1;
-        const shrink = this.size !== 'auto' ? 0 : 1;
-
-        return `${grow} ${shrink} ${this.size}`;
+        const isAuto = this.size === 'auto' && !this.dragSize;
+        const grow = !isAuto ? 0 : 1;
```

Comment:
Why you need a second variable for one and the same value?

Reply:
Do you mean size and dragSize? dragSize represents the size in pixels while you are still dragging the splitter between two panes. Once you drop it that value is converted to %  and applied to the actual size. The reason behind this is that the conversion px to % tends to lose precision and we don't want to have to lose precision on each mouse move while the user is dragging. So we use dragSize as a temporary value while dragging and do the conversion to % only once on move end.

Reply:
I meant `grow` and `shrink`.